spec/cc/engine/analyzers/node_translator_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "spec_helper"
+require "cc/engine/analyzers/node_translator"
+
+RSpec.describe(CC::Engine::Analyzers::NodeTranslator) do
```

Comment:
`NodeTranslator` is overloaded IMO, because `CC::Parser::NodeTranslator` exists. I liked this better when it was named `SexpBuilder`, or something that indicates that it's `Node -> Sexp` (because `NodeTranslator` implies `Node -> Node` to me). Can you describe a little bit what led you to this name?